src/index.ml

```diff
@@ -118,7 +118,7 @@ module Make (K : Key) (V : Value) (IO : IO) = struct
 
   let index_path root = root // "index" // "index"
 
-  let page_size = Int64.mul entry_sizeL 1_000L
+  let page_size = Int64.mul entry_sizeL 1L
```

Comment:
This seems like the opposite of batch reading, doesn't it hurt the performance of the operations using `iter_io` ?

Reply:
do we use `iter_io` in the benchmarks? :-)

But yes, you are right this is used only in iter_io so that should not change.

Reply:
No we don't, `iter_io` is used on startup, and to sync readonly instances.
I'll add some more benchs.